tasks/main.yml

```diff
@@ -12,3 +12,17 @@
     baseurl: https://depot.galaxyproject.org/yum/el/$releasever/$basearch/
     gpgcheck: no
   when: ansible_os_family == "RedHat"
+
+- name: Add slurm-drmaa (Debian)
+  block:
+    - name: Ensure that source GPG key is known
+      ansible.builtin.apt_key:
+        keyserver: hkp://keys.openpgp.org
```

Comment:
```suggestion
        keyserver: hkp://keyserver.ubuntu.com
```

Reply:
> Nate's key is not present in the ubuntu keyserver

ahhh jeez.

Reply:
It's here: http://keyserver.ubuntu.com/pks/lookup?search=18381AC8832160AF&fingerprint=on&op=index

Perhaps someone just submitted it?

Reply:
Seems not valid as mentionned in my post:
`new key but contains no user ID - skipped`